website/sidebars.json

```diff
@@ -5,7 +5,8 @@
       "nos-client/getting-started",
       "nos-client/api",
       "nos-client/implementation-example",
-      "nos-client/features"
+      "nos-client/features",
+      "nos-client/troubleshooting.md"
```

Comment:
@DalderupMaurice Will this work with the `.md` extension here?  I see the other lines don't include it, but maybe it's irrelevant.

Reply:
Oh, no, you are right, completely my fault. Thanks for spotting that out! Fixed